### ISSUE TITLE
RX - Remove Tabs & Move "We can't refill"

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#7434]
## Description 
As a VA, I want to update the RX Sort/Filter function, so that it may benefit the Veteran users more. 

#### 1. Address screen component IA
-  [ ] Remove tabs of All, Pending, and Tracking under the Start refill request 
-  [ ] Move "We can't refill...in the app" to below Start refill request button
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWJ3QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3be27d3f40684a1aaa357008669a4096f3c688fd/image.png)

**Accessibility Requirements**
- [ ] Screen reader accessible 

## Notes & Open Questions
- [Figma prototype ](https://www.figma.com/proto/Si4LF4t8VOv8emdCovv3U3/%F0%9F%94%8D-Prescriptions-2.0---Working---VAMobile?page-id=1499%3A15245&type=design&node-id=1499-15407&viewport=812%2C486%2C0.31&t=gjAhJIZN0xMTxh98-1&scaling=min-zoom&starting-point-node-id=1499%3A15407)
- [Figma Prototype instructions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/medication/Design/List%20enhancements%20Q4%202023.md)
-During 11/30 Health team meeting, engineering asked to split the ticket 7355 into 2 tickets
   1.) Remove tabs and moving the We can't refill section
   2.) New sort/filter page / functionality
- Any internal/external dependencies? Yes - test accounts with the necessary statuses to test 
- Test accounts needed? Yes
- Does this require QA? Yes

**Comment by alexandec** [comment]
Some questions:

 - If we remove the tabs in this ticket, but the new sort/filter functionality won't be in place yet, won't the users lose that functionality until the other ticket is completed?
 - Do we need a feature flag for these changes?
 - Do we have reworked analytics covering the new structure with no tabs and everything happening in "Filter and sort"?

**Comment by Sparowhawk** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VA-Mobile-60f1a34998bc75000f2a489f/issues/department-of-veterans-affairs/va-mobile-app/7434?planning-poker) with Zenhub @Sparowhawk @alexandec

**Comment by alexandec** [comment]
Added my estimate. I was thinking this one would be smaller than #7355 since this only covers removing the tabs and moving that collapsible message

**Comment by alexandec** [comment]
@DJUltraTom QA note for this one, Binny merged the PR accidentally, but the target branch was `feature/chris-rx-filter-improvements` rather than develop, so it got merged into that branch instead. You can just do QA on that branch. If you find any issues, I'll address them in the [second ticket](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7355) instead.

**Comment by DJUltraTom** [comment]
RX revision test cases look good, ran through fine, QA approves

**Comment by DJUltraTom** [comment]
waiting on 7355 to validate on dev branch

**Comment by DJUltraTom** [comment]
waiting on all items in [7487](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/7487) to be completed before final validation and sign off can occur

**Comment by alexandec** [comment]
@DJUltraTom this is now merged into develop and ready for final testing there

**Comment by DJUltraTom** [comment]
verified this is looking good on both iOS and android on develop.
Closing as "Deployed"
